components/match2/wikis/sideswipe/match_group_input_custom.lua

```diff
@@ -0,0 +1,532 @@
+				opponent.status = _STATUS_HAS_SCORE
+				isScoreSet = true
+			elseif Table.includes(_ALLOWED_STATUSES, opponent.score) then
+				opponent.status = opponent.score
+				opponent.score = -1
+			end
+
+			--set Walkover from Opponent status
+			args.walkover = args.walkover or _STATUS_TO_WALKOVER[opponent.status]
+
+			opponents[opponentIndex] = opponent
+
+			-- get players from vars for teams
+			if opponent.type == 'team' and not Logic.isEmpty(opponent.name) then
+				args = matchFunctions.getPlayers(args, opponentIndex, opponent.name)
+			end
+		end
+	end
+
+	--set resulttype to 'default' if walkover is set
+	if args.walkover then
+		args.resulttype = 'default'
+	end
+
+	local autofinished = String.isNotEmpty(args.autofinished) and args.autofinished or true
+	-- see if match should actually be finished if score is set
+	if isScoreSet and Logic.readBool(autofinished) and not Logic.readBool(args.finished) then
+		local currentUnixTime = os.time(os.date('!*t'))
+		local lang = mw.getContentLanguage()
+		local matchUnixTime = tonumber(lang:formatDate('U', args.date))
+		local threshold = args.dateexact and 30800 or 86400
+		if matchUnixTime + threshold < currentUnixTime then
+			args.finished = true
+		end
+	end
+
+	-- apply placements and winner if finshed
+	local winner = tostring(args.winner or '')
+	if Logic.readBool(args.finished) then
+		local placement = 1
+		local lastScore
+		local lastPlacement = 1
+		local lastStatus
+		-- luacheck: push ignore
+		for opponentIndex, opponent in Table.iter.spairs(opponents, p._placementSortFunction) do
+			if opponent.status ~= _STATUS_HAS_SCORE and opponent.status ~= _STATUS_DEFAULT_WIN and placement == 1 then
+				placement = 2
+			end
+			if placement == 1 then
+				args.winner = opponentIndex
+			end
+			if opponent.status == _STATUS_HAS_SCORE and opponent.score == lastScore then
+				opponent.placement = lastPlacement
+			elseif opponent.status ~= _STATUS_HAS_SCORE and opponent.status == lastStatus then
+				opponent.placement = lastPlacement
+			else
+				opponent.placement = placement
+			end
+			args['opponent' .. opponentIndex] = opponent
+			placement = placement + 1
+			lastScore = opponent.score
+			lastPlacement = opponent.placement
+			lastStatus = opponent.status
+		end
+	-- luacheck: pop
+	-- only apply arg changes otherwise
+	else
+		for opponentIndex, opponent in pairs(opponents) do
+			args['opponent' .. opponentIndex] = opponent
+		end
+	end
+	if
+		winner == _RESULT_TYPE_DRAW or
+		winner == '0' or (
+			Logic.readBool(args.finished) and
+			#opponents == 2 and
+			opponents[1].status == _STATUS_HAS_SCORE and
+			opponents[2].status == _STATUS_HAS_SCORE and
+			opponents[1].score == opponents[2].score
+		)
+	then
+		args.winner = 0
+		args.resulttype = _RESULT_TYPE_DRAW
+	elseif
+		Logic.readBool(args.finished) and
+		#opponents == 2 and
+		opponents[1].status ~= _STATUS_HAS_SCORE and
+		opponents[1].status == opponents[2].status
+	then
+		args.winner = 0
+	end
```

Comment:
New wiki, shouldn't have two ways to enter a draw (`winner=draw` and `winner=0`). 
Wiki should pick one and go with it.